### ISSUE TITLE
List all of the reviews that I have written

### ISSUE THREAD

**Issue by jungaretti** [microsoft/vsmarketplace#674]
Is there a page on the marketplace that lists all of the reviews that I have written? For example, [here's how to view all of the reviews you have shared on Apple's App Store](https://www.idownloadblog.com/2020/11/09/see-your-app-store-ratings-and-reviews/). I'd like to see all of my previous reviews to make sure they are accurate.

**Comment by seaniyer** [comment]
@jungaretti 

There is not one at this time. Thanks for the feedback. Just curious, was there a specific use case that prompted the feedback?

- VS Marketplace team

**Comment by jungaretti** [comment]
> Just curious, was there a specific use case that prompted the feedback?

Yes. I was thinking about how influential reviews are. Positive reviews help build trust and bring new users to extensions. I wanted to see all of the reviews I've left and make sure they are accurate.

For example, if I left a poor review on an extension 3 years ago and that extension has improved recently, I'd like to remove it or revise it.
